spec/mem/pointer.md

```diff
@@ -99,13 +99,30 @@ pub struct PointeeInfo {
     pub unpin: bool,
 }
 
+pub struct TupleHeadLayout {
+    /// The offset where the tail starts.
+    /// Also determines the size together with the tail size and padding to satisfy alignment.
+    head_end: Offset,
+
+    /// The minimal alignment of the head.
+    min_align: Align,
```

Comment:
`min_` seems redundant, alignment is always a lower bound (it's always okay to align something more).

Reply:
Maybe call it `head_align`?